internal/flows/uninstall.go

```diff
@@ -83,7 +85,14 @@ func (u *Uninstaller) uninstallDaemons(ctx context.Context) error {
 		}
 
 		ssmClient := awsSsm.NewFromConfig(awsConfig, func(o *awsSsm.Options) {
-			o.RetryMaxAttempts = 6
+			o.Retryer = retry.NewAdaptiveMode(func(ao *retry.AdaptiveModeOptions) {
+				ao.StandardOptions = []func(*retry.StandardOptions){
+					func(so *retry.StandardOptions) {
+						so.MaxAttempts = 6
```

Comment:
what do you think about: attempts: 12 and backoff: 1 minute

add a comment like (or whatever you see fit):

```
intentionally long max backoff and number of retry attempts as we optimize for success of this operation
vs flaky fails during deregistering. we tend to see more connection reset (and the like) errors from the ssm endpoint
```

Reply:
Let's wait for abhays ssm or to merge and update this to use that format